### PR TITLE
[Keyboard] Update preonic/Kjwon15 layout

### PR DESCRIPTION
**Pull request by Kjwon15**
## Description

Make it possible to ctrl+click on mouse layer

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Fix doesn't responding from booting/wake up from sleeping

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/preonic/keymaps/kjwon15/rules.mk [hunk 1]
```diff
@@ -1,2 +1,3 @@
 SRC += muse.c
 KEY_LOCK_ENABLE = yes
+SLEEP_LED_ENABLE = no
```
Comment (fauxpark) [inline keyboards/preonic/keymaps/kjwon15/rules.mk:3]:
```suggestion
```

Comment (Kjwon15) [reply keyboards/preonic/keymaps/kjwon15/rules.mk:3]: Can I ask you why?

Comment (fauxpark) [reply keyboards/preonic/keymaps/kjwon15/rules.mk:3]: It's not needed; none of the Preonic revisions have this set to `yes`.

Comment (Kjwon15) [reply keyboards/preonic/keymaps/kjwon15/rules.mk:3]: So should I remove it?

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**